02_helper_functions/01_topics_mapping/setupParams.m

```diff
@@ -0,0 +1,12 @@
+function [params] = setupParams()
```

Comment:
This is fine but on the long run I think we have to move away those scripts and automatically detect the available topics and params.

Reply:
I think that is in general a nice idea, but the downside would be not catching branch specific topics/params (eg sysid branch that should not be merged to matter, but will be frequently used) as not logged. I guess this could be resolved by simply looking for the struct field in the place they are used.. but not sure which is more efficient. We should discuss.